b2

```diff
@@ -1327,6 +1390,17 @@ def download_file_by_name(args):
 
 
 class ConsoleTool(object):
+
+    """
+    Implements the commands available in the B2 command-line tool
+    using the B2Api library.
+
+    Responsible for storing cached account data in ~/.b2_account_info,
```

Comment:
I don't think `ConsoleTool` should be responsible for that - other `B2Api` callers should be saved from doing it in their code unless it is necessary.